tasks.go

```diff
@@ -135,6 +138,10 @@ func GetNagiosTasks(tasks chan<- Task, projects []string, basepath string) {
 			tasks <- GetNagiosHistoricalData(p, pod, outFor, errOutFor)
 		}
 	}
+
+	if !foundANagiosPod {
+		fmt.Println("A Nagios pod could not be found in any project. For a more thorough analysis, please ensure Nagios is running in all RHMAP projects.")
```

Comment:
We should not print to stdout, or it won't be included in the `dump.log`.
Would it make sense to make it a task error?